### PR TITLE
fix: Fix .env path for test scripts

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

**Review by chadwhitacre — APPROVED** [review]
Good catch.

**Comment by BYK** [comment]
> ././_lib.sh: line 19: /home/runner/work/onpremise/onpremise/install/.env: No such file or directory

Apparently not enough.

**Comment by chadwhitacre** [comment]
> Apparently not enough.

Where are you seeing that? I see an error on CI for the latest commit, but the initial commit is green afaict. No?

**Comment by chadwhitacre** [comment]
Should we add a test for this? How did this come up in the first place?

File: install/_lib.sh [hunk 1]
```diff
@@ -6,14 +6,13 @@ log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
...
-if [[ "$(basename $0)" = "install.sh" ]]; then
+if [[ "$(basename $0)" = "install.sh" || "$(basename $0)" = "test.sh" ]]; then
   cd "$(dirname $0)/install/"
 else
   cd "$(dirname $0)"  # assume we're a *-test.sh script
 fi
 
-_ENV="$(realpath ./.env)"
-
+_ENV="$(realpath ../.env)"
```
Comment (chadwhitacre) [inline install/_lib.sh]: This is still a problem, though. Right? Don't we need the first commit that varies this with the base dir?

**Comment by BYK** [comment]
>Should we add a test for this? How did this come up in the first place?

We definitely should. I saw this here by chance: https://github.com/getsentry/onpremise/runs/2404982826?check_suite_focus=true#step:4:683

**Review by chadwhitacre — CHANGES_REQUESTED** [review]

File: install/_lib.sh (hunk 1 above)
Comment (BYK) [reply install/_lib.sh]: Nope, the logic there was valid :D

Comment (chadwhitacre) [reply install/_lib.sh]: Got it, because dirname of $0 is `.../install/` if we assume it's a `*-test.sh` script. 👍

**Review by chadwhitacre — APPROVED** [review]

**Pull request merged**